src/fragments/lib/auth/flutter/getting_started/60_developerPreview.mdx

```diff
@@ -0,0 +1,9 @@
+<Callout warning>
+
+Web and desktop support for the Amplify Flutter Auth category is now in developer preview!  See the documentation for [project setup](/lib/project-setup/create-application) and the [Auth category](/lib/auth/getting-started) for more information about getting started.
```

Comment:
I think we use this callout in 2 places, in the getting started section and within the auto category. For getting started it looks great, for the Auth category since the sentence has a link to Auth category from the user's mind it might seem like the link is broken. Should we have a slightly different worded callout for this section?